### ISSUE TITLE
this.openLink error on clicking link to check for server options

### ISSUE THREAD

**Issue by lukestokes** [greymass/eos-voter#114]
When presented with this screen:

> Not sure what server to use?
> Click here to view a list of options.

I get this error when clicking the link:

```
Uncaught TypeError: this.openLink is not a function
    at onClick (Connection.js:106)
    at Object.<anonymous> (react-dom.production.min.js:15)
    at Object.invokeGuardedCallback (react-dom.production.min.js:16)
    at Object.invokeGuardedCallbackAndCatchFirstError (react-dom.production.min.js:16)
    at B (react-dom.production.min.js:20)
    at T (react-dom.production.min.js:22)
    at N (react-dom.production.min.js:22)
    at I (react-dom.production.min.js:21)
    at F (react-dom.production.min.js:24)
    at D (react-dom.production.min.js:24)
```

**Comment by aaroncox** [comment]
What version? I thought I had this fixed in 0.1.7.

**Comment by lukestokes** [comment]
Ah, sorry. I installed 1.0.6 this morning thinking it was the latest version, but I see 1.0.7 came out 5 hours ago. I'll upgrade.
